packages/@vuepress/theme-default/components/DropdownLink.vue

```diff
@@ -109,6 +109,11 @@ export default {
   .dropdown-title
     display block
     font-size 0.9rem
+    font-family inherit
+    cursor inherit
+    padding inherit
+    line-height 1.4rem
+    outline none
```

Comment:
We must keep outline property

Reply:
I removed the ‘outline none’, hope the accessibility can be solved by a more elegant way in the future.